components/operators-detail/documentation/documents-by-operator.js

```diff
@@ -15,7 +15,7 @@ import { HELPERS_DOC } from 'utils/documentation';
 // Components
 import DocCard from 'components/ui/doc-card';
 import DocCardUpload from 'components/ui/doc-card-upload';
-import DocumentsByFMU from './documents-by-fmu';
+import DocumentsByFMU from './documents-by-fmu.js';
```

Comment:
you don't need to add .js right?

Reply:
`eslint` asks for extension when the import is relative. I guess I could disable that rule, too. Check https://github.com/Vizzuality/otp-portal/blob/feature/operators-detail-format/.eslintrc#L49 and let's agree on a common rule/solution?

Reply:
Yes, disable it

Reply:
Done, you can merge if you want 👍